src/gui/plugins/mouse_drag/MouseDrag.qml

```diff
@@ -23,23 +23,72 @@ GridLayout {
   columns: 8
   columnSpacing: 10
   Layout.minimumWidth: 350
-  Layout.minimumHeight: 200
+  Layout.minimumHeight: 300
   anchors.fill: parent
   anchors.leftMargin: 10
   anchors.rightMargin: 10
 
+  // Maximum value for GzSpinBoxes
+  property int maxValue: 1000000
+
+  // Precision of the GzSpinBoxes in decimal places
+  property int decimalPlaces: 1
+
+  // Step size of the GzSpinBoxes
+  property double step: 10.0
+
   Text {
     Layout.columnSpan: 8
     id: rotationText
     color: "dimgrey"
-    text: qsTr("Ctrl+Left-Click: rotation")
+    text: qsTr("Rotation (Ctrl+Left-Click)")
+  }
+
+  Label {
+    Layout.columnSpan: 2
+    horizontalAlignment: Text.AlignRight
+    id: rotStiffText
+    color: "dimgrey"
```

Comment:
nit: The `dimgrey` makes it look like these options are disabled. Can we just use the same color as used for " Apply force to center of mass"?

Reply:
Done in 3a6c617